Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm

```diff
@@ -2477,4 +2477,58 @@ InstanceMethodSwizzler swizzler(PAL::getWTWritingToolsClass(), @selector(schedul
     TestWebKitAPI::Util::run(&finished);
 }
 
+TEST(WritingTools, SmartReplyTest)
```

Comment:
If we can rewrite smart replies, can you add a test that does that as well?

Reply:
you can't rewrite them

Reply:
maybe call the test `SmartReplyWithInsertedSpace`? I know I originally named it 😅